components/book/Form.tsx

```diff
@@ -5,37 +5,45 @@ import { Book } from "../../interfaces/Book";
 import Link from "next/link";
 
 interface Props {
-	book: Book;
+	book?: Book;
 }
 
 export const Form: FunctionComponent<Props> = ({ book }) => {
-	const handleDelete = () => {
+
+  const handleDelete = () => {
 		if (window.confirm("Are you sure you want to delete this item ?")) {
 			try {
 				fetch(`${book["@id"]}`, { method: "delete" });
 			} catch (error) {
-				alert("Error when delete element");
+				alert("error when delete element");
 				console.error(error);
 			}
 		}
 	};
 
-	const defaultValues = book["@id"]
-		? {
-				isbn: "",
-				title: "",
-				description: "",
-				author: "",
-				publicationDate: "",
-				reviews: "",
-		  }
-		: { ...book };
+	const defaultValues = () => {
+    console.log("book", book)
+    if (book["@id"] !== undefined) {
```

Comment:
```suggestion
    if (book) {
```

The `book` prop is optional, and will not be provided for the creation form.